telegram/_chat.py

```diff
@@ -910,10 +910,12 @@ def __init__(
             for arg in _deprecated_attrs:
                 if (val := object.__getattribute__(self, arg)) is not None and val != ():
                     warn(
-                        f"The argument `{arg}` is deprecated and will only be available via "
-                        "`ChatFullInfo` in the future.",
+                        PTBDeprecationWarning(
+                            "NEXT.VERSION",
```

Comment:
Just as reminder: In cas this doesn't make it into the same release as #4260, this needs updating